lib/rules/media-query-no-invalid/index.mjs

```diff
@@ -143,6 +159,31 @@ const rule = (primary) => {
 	};
 };
 
+/**
+ * @param {import('@csstools/media-query-list-parser').MediaQuery|import('@csstools/media-query-list-parser').GeneralEnclosed} node
+ * @param {Parameters<Rule>[1]} secondaryOptions
+ * @returns {boolean}
+ */
+function shouldIgnoreNode(node, secondaryOptions) {
+	let ignored = false;
+
+	if (secondaryOptions?.ignoreFunctions) {
+		node.walk(({ node: childNode }) => {
+			if (!isFunctionNode(childNode)) {
```

Comment:
question

If you can have multiple child nodes and non-functional nodes may be encountered, couldn't we have an ignored function following a non-functional node?
In that case, wouldn't it return too early?
i.e. before `ignored` can be set to true at the next iteration

Reply:
I couldn't, my question was probably unfounded.

---

I found something weird though; in the previous conversation it was said that if one function is ignored, everything must be ignored.
I tried

```
		{
			code: '@media (getMinWidth <= width <= foo()) {}',
		},
```
and it didn't pass with `config: [true, { ignoreFunctions: ['/^get.*$/'] }],`
is it by design?

Reply:
I couldn't, my question was probably unfounded.

Reply:
Does it pass with `@media (getMinWidth() <= width <= foo()) {}`?
(I haven't tried)

Reply:
> (I haven't tried)

If you don't know yet, it might be useful if you plan to make a lot of reviews:

```sh
git fetch upstream pull/8060/head:add-ignore-functions-to-media-query-no-invalid && git checkout add-ignore-functions-to-media-query-no-invalid && git pull upstream pull/8060/head:add-ignore-functions-to-media-query-no-invalid
```

You gotta replace `upstream` by what you are using for the remote of `stylelint/stylelint`.

> Does it pass with `@media (getMinWidth() <= width <= foo()) {}`?

It passes.